commit-status-publisher-server/src/main/java/jetbrains/buildServer/commitPublisher/github/GetJiraTickets.java

```diff
@@ -0,0 +1,58 @@
+package jetbrains.buildServer.commitPublisher.github;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import com.intellij.openapi.diagnostic.Logger;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+public class GetJiraTickets {
+	private static final Logger LOG = Logger.getInstance(GetJiraTickets.class.getName());
+	private final static String jira_link = "https://ankiinc.atlassian.net/browse/";
+	private final static String od = "od\\d*";
+	private final static String cozmo = "coz\\d*";
+	private final static String bi = "bi\\d*";
+	
+	@NotNull
+	public static List<String> getListJiraTickets(@NotNull String branch_name){
+		
+		List<String> list_jira_link = new ArrayList<String>();
+		String name = branch_name.toLowerCase().replaceAll("[^\\da-z]", "");

```

Comment:
Does this work on the branch name or the github title?  We might want to change some of the branch_name or title variables